engine/runner-common/src/main/java/module-info.java

```diff
@@ -0,0 +1,12 @@
+module org.enso.runner.common {
+  requires commons.cli;
+  requires org.apache.commons.io;
+  requires org.enso.editions;
+  requires org.enso.engine.common;
+  requires org.enso.librarymanager;
+  requires org.enso.pkg;
+  requires org.slf4j;
+  requires scala.library;
```

Comment:
Áááágr:
```
engine/runner-common/src/main/scala/org/enso/runner/common/CompilerBasedDependencyExtractor.scala
engine/runner-common/src/main/scala/org/enso/runner/common/ProfilingConfig.scala
```
Just two classes away from getting rid of another Scala dependency - alas, not: the scal would get in the list of dependencies transitively anyway.